src/story.ts

```diff
@@ -106,8 +106,20 @@ export type StoryContext<
   canvasElement: HTMLElement;
 };
 
+export type StepFunction<TFramework extends AnyFramework = AnyFramework, TArgs = Args> = (
+  stepName: string,
+  substeps: PlayFunction<TFramework, TArgs>
```

Comment:
I don't like the name of this. I suggest `callback`.

Reply:
Hmm, don't love either of those either. `callback` implies it'll run later on. `play` is confusing, no?

Reply:
I don't think `play` is confusing at all. It makes sense in a way. I updated my PR to use `name` and `play` as argument names.

Reply:
Implementing the playContext is a little more tricky with the cyclic reference, but it works: https://github.com/storybookjs/storybook/blob/26ec0e6e3d786404cb24ef089b3f9e71ad22fba8/lib/preview-web/src/StoryRender.ts#L60-L64